bin/fin

```diff
@@ -2855,6 +2856,68 @@ config_generate ()
 	fi
 }
 
+create_site ()
+{
+	while true; do
+		read -p "Please enter the name of your project (all one word, no spaces or slashes): " projectName
+		#todo, validate this input
+		break
+	done
+	echo ''
+
+	echo -e "What would you like to install?"
+	echo -e "${green}[1]${NC} Drupal 7"
+	echo -e "${green}[2]${NC} Drupal 8"
+	echo -e "${green}[3]${NC} Wordpress"
+	echo -e ""
+	while true; do
+		read -p ": " newSiteVersion
+		case ${newSiteVersion} in
+			1 )
+				newHumanReadableVersion="Drupal 7"
+				newSiteRepoName="drupal7"
+				break
+				;;
+			2 )
+				newHumanReadableVersion="Drupal 8"
+				newSiteRepoName="drupal8"
```

Comment:
Could you also make branch to be `${FIN_BRANCH}`

`git clone -b ${FIN_BRANCH} ${target_repo} ${project_name}`

This is for dev testing purposes.